Pomodoro.Core/Interfaces/IServices/IStatisticsService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Pomodoro.Core.Models.Statistics;
+
+namespace Pomodoro.Core.Interfaces.IServices
+{
+    public interface IStatisticsService
+    {
+        Task<DailyStatistics> GetDailyStatisticsAsync(int userId, DateOnly day);
```

Comment:
probably userId will be a GUID

Reply:
Should I also change the type of Ids (from int to GUID) for our Entities?

Reply:
> Should I also change the type of Ids (from int to GUID) for our Entities?

Yes, I think it would make sense

Reply:
Ok, I'll do that along with other fixes in DAL while working on the `add-repository` branch.